src/components/Offices/CompanyRegistry/index.js

```diff
@@ -76,8 +79,13 @@ function CompanyRegistration({ registration }) {
       {' '}
       {registration.registration.editableByRegistrar.toString()}
       <div className={styles.buttonWrapper}>
-        <Button primary medium red onClick={unregister}>
-          Unregister company
+        <Button
+          primary
+          medium
+          red
+          onClick={() => unregisterCompany(registration.entity_id)}
+        >
+          Unregister company hard
```

Comment:
`soft` vs `hard` aren't very good names, let's not show it to the user. Let's leave simple `Unregister company` here.